src/UCTNodePointer.cpp

```diff
@@ -25,18 +25,34 @@
 
 #include "UCTNode.h"
 
+std::atomic<size_t> UCTNodePointer::m_tree_size = { 0 };
```

Comment:
Feels a bit weird to have this here instead of UCTSearch but it's probably a bit more convenient I guess.

Though, code like here https://github.com/gcp/leela-zero/pull/1741/files#diff-70aab7683fe64f9428b8bc47d1d183c0R179
would look more natural if it was part of UCTSearch.

Reply:
I don't see a perfect solution and see some good reasons to keep it on either class, so I decided to keep it where the implementation is easier. I'm open to suggestions on how to improve it however.